TypeRaceAir/race.py

```diff
@@ -0,0 +1,63 @@
+import typing
+
+
+class Race():
+
+    def __init__(self, sentece: str, players: typing.List[Player]):
+        self.sentece = sentece
+        self.players = players
+
+
+class Player():
+    pass
+
+
+class PlayerRaceResult():
+    def __init__(self, race: Race, player: Player, submit_timestamp: float, submit_sentence: str):
```

Comment:
If there are too many arguments in a single line, consider using the following syntax
```suggestion
    def __init__(self,
        race: Race,
        player: Player,
        submit_timestamp: float,
        submit_sentence: str
        ):
```